spec/components/schemas/Organization.yaml

```diff
@@ -67,7 +61,7 @@ properties:
     description: Invoice will use this time zone to display time otherwise UTC will be used. Example "America/New_York"
     type: string
     maxLength: 50
-  reportCurrency:
+  reportingCurrency:
```

Comment:
shouldn't it be consisted with https://github.com/Rebilly/RebillyReportsAPI/blob/master/spec/components/schemas/Organization.yaml